extensions/dacpac/src/test/testDacFxService.ts

```diff
@@ -58,103 +58,104 @@ export class DacFxTestService implements mssql.IDacFxService {
 			success: true,
 			errorMessage: '',
 			deploymentOptions: {
-				ignoreTableOptions: false,
-				ignoreSemicolonBetweenStatements: false,
-				ignoreRouteLifetime: false,
-				ignoreRoleMembership: false,
-				ignoreQuotedIdentifiers: false,
-				ignorePermissions: false,
-				ignorePartitionSchemes: false,
-				ignoreObjectPlacementOnPartitionScheme: false,
-				ignoreNotForReplication: false,
-				ignoreLoginSids: false,
-				ignoreLockHintsOnIndexes: false,
-				ignoreKeywordCasing: false,
-				ignoreIndexPadding: false,
-				ignoreIndexOptions: false,
-				ignoreIncrement: false,
-				ignoreIdentitySeed: false,
-				ignoreUserSettingsObjects: false,
-				ignoreFullTextCatalogFilePath: false,
-				ignoreWhitespace: false,
-				ignoreWithNocheckOnForeignKeys: false,
-				verifyCollationCompatibility: false,
-				unmodifiableObjectWarnings: false,
-				treatVerificationErrorsAsWarnings: false,
-				scriptRefreshModule: false,
-				scriptNewConstraintValidation: false,
-				scriptFileSize: false,
-				scriptDeployStateChecks: false,
-				scriptDatabaseOptions: false,
-				scriptDatabaseCompatibility: false,
-				scriptDatabaseCollation: false,
-				runDeploymentPlanExecutors: false,
-				registerDataTierApplication: false,
-				populateFilesOnFileGroups: false,
-				noAlterStatementsToChangeClrTypes: false,
-				includeTransactionalScripts: false,
-				includeCompositeObjects: false,
-				allowUnsafeRowLevelSecurityDataMovement: false,
-				ignoreWithNocheckOnCheckConstraints: false,
-				ignoreFillFactor: false,
-				ignoreFileSize: false,
-				ignoreFilegroupPlacement: false,
-				doNotAlterReplicatedObjects: false,
-				doNotAlterChangeDataCaptureObjects: false,
-				disableAndReenableDdlTriggers: false,
-				deployDatabaseInSingleUserMode: false,
-				createNewDatabase: false,
-				compareUsingTargetCollation: false,
-				commentOutSetVarDeclarations: false,
-				blockWhenDriftDetected: false,
-				blockOnPossibleDataLoss: false,
-				backupDatabaseBeforeChanges: false,
-				allowIncompatiblePlatform: false,
-				allowDropBlockingAssemblies: false,
-				dropConstraintsNotInSource: false,
-				dropDmlTriggersNotInSource: false,
-				dropExtendedPropertiesNotInSource: false,
-				dropIndexesNotInSource: false,
-				ignoreFileAndLogFilePath: false,
-				ignoreExtendedProperties: false,
-				ignoreDmlTriggerState: false,
-				ignoreDmlTriggerOrder: false,
-				ignoreDefaultSchema: false,
-				ignoreDdlTriggerState: false,
-				ignoreDdlTriggerOrder: false,
-				ignoreCryptographicProviderFilePath: false,
-				verifyDeployment: false,
-				ignoreComments: false,
-				ignoreColumnCollation: false,
-				ignoreAuthorizer: false,
-				ignoreAnsiNulls: false,
-				generateSmartDefaults: false,
-				dropStatisticsNotInSource: false,
-				dropRoleMembersNotInSource: false,
-				dropPermissionsNotInSource: false,
-				dropObjectsNotInSource: false,
-				ignoreColumnOrder: false,
+				ignoreTableOptions: { value: false, description: sampleDesc },
```

Comment:
can you move these options changes to a separate PR from the new publish options dialog changes so that it'll be easier to review? And it'll make it easier to roll back the separate changes just in case.

Reply:
Done